config.json

```diff
@@ -343,34 +346,37 @@
         "slug": "hello-world",
         "name": "Hello World",
         "uuid": "2c3902df-93e1-48d1-829d-4daa93a1f06f",
-        "practices": [],
-        "prerequisites": [],
-        "difficulty": 1,
-        "topics": [
+        "practices": [
           "strings"
```

Comment:
Let's keep hello world out if this, it's not really meant to practice anything.

Reply:
Done